lms/resources/_js_config/file_picker_config.py

```diff
@@ -4,6 +4,8 @@ class FilePickerConfig:
     @classmethod
     def blackboard_config(cls, context, request, application_instance):
         """Get Blackboard files config."""
+        if not application_instance.product == application_instance.Product.BLACKBOARD:
+            return {}
```

Comment:
Related to https://github.com/hypothesis/lms/pull/3356 but if returning None here it breaks this branch.